compiler/src/yul/runtime/functions/data.rs

```diff
@@ -66,20 +66,23 @@ pub fn load_data_string() -> yul::Statement {
 pub fn mcopys() -> yul::Statement {
     function_definition! {
         function mcopys(mptr, sptr, size) {
-            (let offset := 0)
-            (for { } (lt((add(offset, 32)), size)) { }
+            (let mptr_offset := 0)
+            (let sptr_offset := 0)
+            (for { } (lt((add(mptr_offset, 32)), size)) { }
```

Comment:
see if this test expression can be cleaned up at all for each of the copying functions